levenshtein-ffi.gemspec

```diff
@@ -6,24 +6,27 @@
 # stub: ext/levenshtein/extconf.rb
 
 Gem::Specification.new do |s|
-  s.name = "levenshtein-ffi"
+  s.name = "levenshtein-ffi".freeze
```

Comment:
nit: If you add the magic comment `# frozen_string_literal: true` to the top of the file, we don't need to add `.freeze` everywhere, which might make reviewing the diff slightly easier.

Reply:
The gem is set up using Jeweler, which generates this file. So unfortunately I don't think there's a way for us to add the magic comment.